package.json

```diff
@@ -329,6 +329,7 @@
     "bull": "^4.2.0",
     "chardet": "^2.0.0",
     "cheerio": "^1.0.0-rc.10",
+    "classnames": "^2.5.1",
```

Comment:
I'm guessing this was meant to be added as a frontend dependency, and not a backend dependency

Reply:
Moved to frontend, thanks!